Исправлен путь для получения пользователя по Id

evgeniiruchko opened the pull request:
Добавлен UserController с получением данных о пользователе по ID и получением активных бонусов по ID пользователя.

В UserService добавлен метод получения активных бонусов.
В настройках исправлена синтаксическая ошибка

evgeniiruchko committed:
Добавлен UserController с получением данных о пользователе по ID и получением активных бонусов по ID пользователя.
В UserService добавлен метод получения активных бонусов.
В настройках исправлена синтаксическая ошибка

AdrenalinV commented on src/main/java/ru/gb/lk_loyality/controllers/UserController.java:
```diff
@@ -0,0 +1,32 @@
+
+    private final UserService userService;
+
+    @GetMapping("/{id}")
```
Тут наверное нужно типа /user/{Id}

evgeniiruchko replied:
переделал

evgeniiruchko committed:
Добавлен UserController с получением данных о пользователе по ID и получением активных бонусов по ID пользователя.
В UserService добавлен метод получения активных бонусов.
В настройках исправлена синтаксическая ошибка
